docs/pages/access-controls/idps/saml-attribute-mapping.mdx

```diff
@@ -0,0 +1,194 @@
+---
+title: SAML IdP Attribute Mapping
+description: How to map user attributes to custom SAML response
+h1: SAML Attribute Mapping
+---
+
+Attribute mapping configures Teleport SAML Identity Provider to assert custom user attributes in SAML response
+The Teleport SAML IdP supports three configurable fields for attribute mapping:
+- `name`: Name of the outgoing attribute. Required. Name should be unique across attribute mapping.
+- `value`: Value defined using a predicate expression, which can reference Teleport usernames, roles or traits. Required.
```

Comment:
Nit: Should we link out "predicate expression" to the [predicate language spec](https://goteleport.com/docs/reference/predicate-language/)?